### ISSUE TITLE
Find dialog combobox uses serif fonts for languages with no monospace fonts available

### ISSUE THREAD

**Issue by tomasz1986** [notepad-plus-plus/notepad-plus-plus#6214]
### Description of the Issue

After https://github.com/notepad-plus-plus/notepad-plus-plus/commit/c35a2b57085a6587b9b356c8680f6d138d5136d6, Notepad++ uses serif fonts to display text in Find dialog for languages like Korean, for which no monospace fonts are available in Windows.

Before the change, sans-serif fonts were used in the dialog. After the change, the default serif font is used, which makes the text blurry, at least in the case of Korean, because the default Korean serif font is not suitable to be used in such small sizes (since its primary use is for printing).

To better understand the problem, especially for those who cannot read Korean, the situation is the same as if you were using Times New Roman to display latin text, which by itself is a good font - for printing, not for the screen, and especially not as a code editor font.

My suggestion would be to either allow customisation of the Find dialog combobox font, or at least use sans-serif fonts for languages where monospace fonts are unavailable.

### Steps to Reproduce the Issue

The steps are for Windows 7.

1.  Go to "Control Panel" - "Region and Language" - "Administrative".
2. Open "Change system locale" and set "Current system locale" to "Korean (Korea)".
3. Restart Windows, start Notepad++, and open the Find dialog.

### Expected Behavior

Notepad++ 7.7.1 - the default sans-serif font is used, and the text is crispy and easy to read.

![2019-10-11-21-41-30-1953125](https://user-images.githubusercontent.com/5626656/66709554-b98c2d80-eda1-11e9-9e5b-8463a0b4e70e.png)

### Actual Behavior

Notepad++ 7.8 - the default serif font is used, and the text is blurry and difficult to read.

![2019-10-11-21-42-01-4218750](https://user-images.githubusercontent.com/5626656/66709557-ce68c100-eda1-11e9-9ca8-804fb76b7330.png)

### Debug Information

Notepad++ v7.8   (32-bit)
Build time : Oct  7 2019 - 00:58:16
Path : npp.7.8.bin\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
OS Name : Windows 7 Enterprise (64-bit) 
OS Build : 7601.0
Plugins : mimeTools.dll NppConverter.dll NppExport.dll

**Comment by sasumner** [comment]
@Thdub said

>font is displayed strangely in the "find" textbox

Screenshot?

**Comment by Thdub** [comment]
![16-10-2019_23h20m51s](https://user-images.githubusercontent.com/37347541/66959915-b5933080-f06b-11e9-9076-dfcd6720391b.png)
By strangely I mean find/replace is the only place I using a serif font, and I find it was way easier to read before...Jumped out at me right after updating

**Comment by tomasz1986** [comment]
> I don't think it is language related, cause I have the exact same problem on en-US (Windows Server 2019)
> I think something got somehow "broken" with latest 7.8 update, font is displayed strangely in the "find" textbox

Yes, the font has got changed to "monospace", while in the past it was "sans-serif". It will look different regardless of the language. The default "monospace" font in Windows for European languages is "Courier New", which is likely the one you are seeing above, in the screen shot from Notepad 7.8. The problem I described is the fact that Windows ships with no "monospace" fonts for many languages, which makes Notepad++ use "serif" fonts in their place.

Don Ho already stated in the forums that he would make this change optional in the future versions (https://community.notepad-plus-plus.org/post/47784).
